project.clj

```diff
@@ -90,7 +103,8 @@
                  [puppetlabs/i18n "0.8.0"]                            ; Internationalization library
                  [redux "0.1.4"]                                      ; Utility functions for building and composing transducers
                  [ring/ring-core "1.6.0"]
-                 [ring/ring-jetty-adapter "1.6.0"]                    ; Ring adapter using Jetty webserver (used to run a Ring server for unit tests)
+                 [ring/ring-jetty-adapter "1.6.0"                     ; Ring adapter using Jetty webserver (used to run a Ring server for unit tests)
+                  :exclusions [commons-io]]
```

Comment:
same. Just trying to understand the reasoning behind this change

Reply:
I was able to remove it